R/assign_job_queue.R

```diff
@@ -94,7 +94,7 @@ get_proc_medians <- function(dir_job_results) {
         dplyr::summarise(
             dplyr::across(
                 dplyr::everything(),
-                \(x) median(x, na.rm = TRUE)
+                \(x) .data$median(x, na.rm = TRUE)
```

Comment:
Here, the error we see for this function is: 
"no visible global function definition for ‘median’"

In this case, we need to add an Rd tag, specifying which median function to use. In this case adding

```
#' @importFrom stats median
```

To the Roxygen2 skeleton above this function should clear the warning